src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerGetTranscriptionTest.java

```diff
@@ -117,7 +117,7 @@ private void addCommentToWorkflow(TranscriptionWorkflowEntity workflowEntity, St
     }
 
     @Test
-    void getTranscriptionNotFound() throws Exception {
+    void whenGetTranscriptionDetailsForNonExistingTranscriptionIsRequested_thenErrorIsReturned() throws Exception {
```

Comment:
Should this be givenANonExistingTranscription_whenGetTranscriptionDetailsIsRequested_thenAnErrorisReturned()

Reply:
hi @mestebanez, you raised a good point. @jackmaloney as raised a similar one [here](https://github.com/mario-paniccia/darts-api-fork/pull/1#discussion_r1417146799). 
Please let me know your preference